storage/src/main/java/tech/pegasys/teku/storage/server/kvstore/dataaccess/KvStoreCombinedDaoAdapter.java

```diff
@@ -546,6 +564,11 @@ public void deleteUnblindedFinalizedBlock(final UInt64 slot, final Bytes32 block
       finalizedUpdater.deleteUnblindedFinalizedBlock(slot, blockRoot);
     }
 
+    @Override
+    public void deleteUnblindedNonCanonicalBlockOnly(final Bytes32 blockRoot) {
```

Comment:
Similar to the `addBlindedFinalizedBlockRaw`, the only real use for this function should be during migration